stable/rethinkdb/templates/rethinkdb-secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  rethinkdb-password: {{ .Values.rethinkdbPassword | b64enc | quote }}
```

Comment:
I don't really think a secure default is of any importance here.
We assume that the person taking care of the databases, will as well take care to generate the secure password and protect the cluster behind a firewall.

As we see on all other databases(MariaDB, MongoDB, Redis, CochroachDB, etc), the default is not to just do the simple and generate a default password due to the problem on `helm upgrade`.